addons/mail/static/src/core/common/message.scss

```diff
@@ -28,6 +28,11 @@
     }
 }
 
+.o-mail-guest-user {
```

Comment:
I don't think this class is useful? The `cursor-pointer` class is only added if `hasAuthorClickable` is `true`. Which should not be the case for guest users.